vision/src/autogluon/vision/predictor/predictor.py

```diff
@@ -165,7 +169,7 @@ def fit(self,
                 # Comparing with `medium_quality_faster_train` it uses faster model but explores more hyperparameters.
                 medium_quality_faster_inference={
                     'hyperparameters': {
-                        'model': Categorical('resnet18_v1b', 'mobilenetv3_small'),
+                        'model': Categorical('resnet18', 'mobilenetv3_small_100', 'resnet18_v1b'),
                         'lr': Categorical(0.01, 0.005, 0.001),
                         'batch_size': Categorical(64, 128),
                         'epochs': Categorical(50, 100),
```

Comment:
Should `epochs` hyperparameter really be searched?

Reply:
Agreed, I don't think we should be searching epochs. Also, why search learning rate via Categorical here but in others it is searched via Real?